### DISCUSSION TITLE
I need help getting verified for student dev pack

### DISCUSSION THREAD

**Discussion by ab-allen** [community/community#17708] [GitHub Education] [answered]
I'm a student at Arizona State University. ID's don't have expiry dates. I've tried submitting class schedules, enrollment verification, student id and keep getting denied. What am I doing wrong? What do I need to submit to prove that I'm a registered student to get  access? Please help me this process.

**Comment by akebu6** [comment]
Did you try submitting your ID? because I got mine today just after applying and using my ID. I got denied the first time when I used my class schedule

**Reply by ab-allen** [reply]
The id for  Arizona State Doesn't work because it doesn't  have an expiration date

**Reply by akebu6** [reply]
Mine also doesn't have an expiration date but it worked. Try using your school's registration form because that worked for one of my friends

**Reply by nurullahsaeem** [reply]
> Mine also doesn't have an expiration date but it worked. Try using your school's registration form because that worked for one of my friends

Did you take a picture of it?

**Comment by mlarasusan** [comment, marked as answer]
Hello [ab-allen](https://github.com/ab-allen), you can find all the details about the documentation in the link below. Thanks :] 

[How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)
